src/dodal/utils.py

```diff
@@ -95,6 +97,8 @@ def __post_init__(self):
 
 
 T = TypeVar("T", bound=AnyDevice)
+D = TypeVar("D", bound=OphydV2Device)
+skip = bool | Callable[[], bool]
```

Comment:
Should: Can we call this something like `skip_type` to avoid confusion?

Reply:
Also maybe should be in PascalCase, similar to most other types and type aliases.

Reply:
> Also maybe should be in PascalCase, similar to most other types and type aliases.

I also agree with this but thought it maybe too opinionated to add. I would rather PascalCase though.

Reply:
:sunglasses: :+1: